### ISSUE TITLE
AMCP Version

### ISSUE THREAD

**Issue by 5opr4ni** [CasparCG/server#784]
![screen shot 2018-02-10 at 11 10 30](https://user-images.githubusercontent.com/20354852/36060931-0f8c8b9c-0e53-11e8-8106-095309ce46f2.png)

**Comment by 5opr4ni** [comment]
This is also the case for AMCP

`>> TLS`

**Comment by ronag** [comment]
cli version depends on #764

**Comment by ronag** [comment]
closing in favor of #764
